backend/consensus/base.py

```
@@ -2084,7 +2097,9 @@ async def handle(self, context):
 
         # Determine the consensus result
         votes_list = list(context.votes.values())
```

Ruff (0.12.2)

Local variable `votes_list` is assigned to but never used

Remove assignment to unused variable `votes_list`

(F841)